website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -262,6 +262,24 @@ Use elements of the same size or resize the inputs. For computer vision applicat
 Disable batching by overriding the `batch_elements_kwargs` function in your ModelHandler and setting the maximum batch size (`max_batch_size`) to one: `max_batch_size=1`. For more information, see
 [BatchElements PTransforms](/documentation/sdks/python-machine-learning/#batchelements-ptransform). For an example, see our [language modeling example](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/inference/pytorch_language_modeling.py).
 
+## Compatible versions
+
+The following table describes how the `RunInference` Model Handlers versions are compatible with
```

Comment:
Wording suggestion: The following table describes the tested configurations of the `RunInference` Model Handlers across different versions of Beam and dependent libraries.  The tested configurations are considered compatible, but other (untested) combinations may also work.